### DISCUSSION TITLE
Code smells detection and refactoring

### DISCUSSION THREAD

**Discussion by konkyr2001** [Ayymoss/HonestSteamBrowser#2] [General]
Dear ayymoss,

I hope this message finds you well. My name is Konstantinos Kyriazopoulos, a 23-year-old individual with a strong background in C# and mostly on creating video games. While exploring GitHub, I came across your project, [BetterSteamBrowser](https://github.com/Ayymoss/BetterSteamBrowser), and was genuinely impressed by its functionality and the innovative approach you've taken in data collection.

I am reaching out to express my interest in contributing to your project. Having worked on various C# projects, with my biggest one being [RocketSpace](https://github.com/konkyr2001/Rocket-Space), I believe I can bring valuable insights and improvements to BetterSteamBrowser. Specifically, I am adept at refining the codebase to achieve a higher standard of cleanliness, readability, and overall efficiency. This, in turn, will contribute to making your program not only faster but also more accessible for future maintenance and upgrades.

I understand the importance of maintaining the integrity of your project, and my goal is to collaborate with you to ensure that any modifications align with your vision. If you're open to it, I would love the opportunity to discuss potential improvements and how we can work together to elevate the project to new heights.

Thank you for considering my proposal, and I appreciate your time in advance. I look forward to the possibility of collaborating with you.

Best Regards,
Konstantinos Kyriazopoulos

**Comment by Ayymoss** [comment]
Yep no problem. I'll need to see the PR first before I can accept any work.
